package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id" : "netuitive.packages.aws.route53",
+    "version" : "0.1.0",
+    "manifest" : {
+      "name" : "Netuitive Packages AWS Route 53 (R53)",
+      "description" : "A set of Netuitive analytics configurations, polices, dashboards, and reports that are used to monitor performance of the elements collected from the AWS Route 53 (R53).",
+      "author" : {
+        "name" : "Netuitive Research",
+        "email" : "research@netuitive.com"
+      },
+      "distro" : {
+        "logo" : "http://assets.app.netuitive.com/pkg/logo/netuitive/banner_logo.png",
+        "download" : "https://github.com/netuitive-community-packages/netuitive-packages-aws-route53/archive/master.zip"
+      },
+      "compatibility" : [ "INFRASTRUCTURE:Route53" ],
```

Comment:
Compatiblility is `AWS:Route53HealthCheck`. This is `<datasource-type>:<element-type>`.

Reply:
@TheConnMan , thanks, I've added the dashboard.
Do we need the computed metric aws.route53.healthecheckstatus.min if there is a policy which shows the same status ("wildcard": "aws.route53.healthcheckstatus.*") and I've set metric meta "STATISTIC": "min" for aws.route53.healthecheckstatus.* ?
I tried to contact @jemassey but his slack account is deactivated.
I've also fixed the "compatibility" line in the package.json